packages/frontend/src/helpers/md5.js

```diff
@@ -0,0 +1,179 @@
+/**
```

Comment:
Tbh, the only check i'd want you to make is to see if we get the same hash as before. If we do, all is well and this can be merged :)

Reply:
perfect, then imho 👍